workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -30,11 +30,6 @@ protected override void OnCreateManager()
 
         protected override void OnUpdate()
         {
-            if (!worker.ConnectionHandler.IsConnected())
-            {
-                return;
-            }
```

Comment:
Why is this moved into the implementation of the `IConnectionHandler`?

Reply:
To prevent the case where you read the same diff again and again. When it says to disconnect it could also clear everything else. However you could have an implementation that clears stuff up in later frames.